[central] intermittent test failure: check-docker-context.sh: No such image

https://github.com/getodk/central/actions/runs/21956300509/job/63421313692?pr=738

```
[check-docker-context.sh] Removing docker image...
Error response from daemon: No such image: sha256:5a39bb8d1f878790b0c024f71a2af7de001c172c57db8d08ae9ba3eeeeb8c7d1
Error: Process completed with exit code 1.
```

--- a/test/check-docker-context.sh
+++ b/test/check-docker-context.sh
@@ -53,9 +53,8 @@ log "Building docker image..."
 iidfile="$(mktemp)"
 (
 docker \
-    buildx build \
+    buildx build --load \
     --iidfile "$iidfile" \
-    --load \
     --no-cache --progress plain --file - . 2>&1 <<EOF
 FROM busybox
 COPY . /build-context

